### PR TITLE
Blog App: Create Data Models

### PR DESCRIPTION
**Pull request by gtekle**
### In this pull request, I have:
  - Created User, Post, Comment, and Like models with their relationships
  - Added db migration for User, Post, Comment, and Like models

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```
Comment (SajjadAhmad14) [inline app/models/like.rb:1]: Please add commands `rails db:create`, `rails db:migrate`, and `npm install or yarn install` in the `Getting Started` section of README for the user to set up the project locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tekle, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by gtekle** [comment]
Hi @SajjadAhmad14 :wave: 
thanks, I'll.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @gtekle , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gtekle** [comment]
Thanks @VICTOR-CHUKWU 🙏

**Pull request merged**